src/clj/rems/application/events.clj

```diff
@@ -16,28 +16,29 @@
    :event/actor UserId
    :application/id s/Int})
 
-(s/defschema ApprovedEvent
+(s/defschema EventWithComment
   (assoc EventBase
+         (s/optional-key :application/comment) s/Str
+         (s/optional-key :event/attachments) [s/Int]))
```

Comment:
should this be `[{:attachment/id s/Int}]` instead?
we might want to return something like `{:attachment/id 1 :attachment/filename "boing.txt"}` from the API, but that can be handled in `enrich-event` perhaps

Reply:
I put it like that now for `:application/forms` for the events. Maybe should do that.

Reply:
yeah, I changed it